### PR TITLE
Adds SAML docs

### PR DESCRIPTION
**Pull request by ttomsu**
@spinnaker/google-reviewers

### CODE REVIEW

File: setup/security/authentication/saml/index.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      </md:IDPSSODescriptor>
+    </md:EntityDescriptor>
+    ```
+
+1. Create a Spinnaker SAML application.
+1. Specify the login URL as `https://localhost:8084/saml/SSO`. Replace "localhost" with Gate's address, if known.
+1. Specify a unique entity ID (we'll use `io.spinnaker:test` in our example).
+1. Enable the users you'd like to have access to your Spinnaker instance.
+
+> TODO: Add halyard commands here when halyard supports SAML
```
Comment (lwander) [inline setup/security/authentication/saml/index.md:71]: Can you file an issue in spinnaker/halyard to link to here?

File: setup/security/authentication/saml/index.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+1. Generate a keystore and key in a new Java Keystore with some password:
+```
+keytool -genkey -v -keystore saml.jks -alias saml -keyalg RSA -keysize 2048 -validity 10000
+```
+2. Create or modify your `gate-local.yml` file to include the following settings:
```
Comment (lwander) [inline setup/security/authentication/saml/index.md:77]: I'm assuming these'll be dropped once the above issue is resolved?

**Review by lwander — APPROVED** [review]

File: setup/security/authentication/saml/index.md (hunk 1 above)
Comment (ttomsu) [reply setup/security/authentication/saml/index.md:71]: https://github.com/spinnaker/halyard/issues/357

File: setup/security/authentication/saml/index.md (hunk 2 above)
Comment (ttomsu) [reply setup/security/authentication/saml/index.md:77]: Yeah - copied here from readme just to have some example, though it probably won't live here for long.

**Pull request merged**